include/solvers/gls_sharp_navier_stokes.h

```diff
@@ -60,165 +61,193 @@ class GLSSharpNavierStokesSolver : public GLSNavierStokesSolver<dim>
+   *parameter: "refine mesh inside radius factor" and "refine mesh outside
+   *radius factor". These factors multiply the radius of the particle to define
+   *the outside and inside radius of the ring\shell.
+   */
+  void
+  refine_ib();
+
+  /**
+   * @brief
+   * Return a bool to define if a cell is cut by an IB particle and the local
+   * DOFs of the cell for later us. If the cell is cut, the function will return
+   * the id of the particle that cut it, else it returns 0.
+   *
+   * @param cell , the cell that we verify whether it is cut or not.
+   *
+   * @param local_dof_indices, a container for the local dof indices used during the function call.
+   *
+   * @param support_points, a mapping of support points for the DOFs.
+   *
+   */
+  std::tuple<bool, unsigned int, std::vector<types::global_dof_index>>
+  cell_cut(const typename DoFHandler<dim>::active_cell_iterator &cell,
+           std::vector<types::global_dof_index> &         local_dof_indices,
+           std::map<types::global_dof_index, Point<dim>> &support_points);
+
+  /**
+   * @brief
+   * Return the cell around a point based on a initial guess of a closed cell
+   * (look in the neighbors of this cell)
+   *
+   * @param cell , The initial cell. We suspect the point of being in one of the neighbours of this cell.
+   *
+   * @param point, The point that we want to find the cell that contains it
+   */
+  typename DoFHandler<dim>::active_cell_iterator
+  find_cell_around_point_with_neighbors(
+    const typename DoFHandler<dim>::active_cell_iterator &cell,
+    Point<dim>                                            point);
+
+  /**
+  * @brief
+  Return a bool that describes  if a cell contains a specific point
+  *
+  * @param cell , The initial cell for which we want to check if the point is inside.
+  *
+  * @param point, The point that we wish to check
+  */
+  bool
+  point_inside_cell(const typename DoFHandler<dim>::active_cell_iterator &cell,
+                    Point<dim> point);
+
+  /**
+   * @brief
+   *Return a vector of cells around a cell including vertex neighbors
+   *
+   * @param cell , The initial cell. we want to know all the cells that share a vertex with this cell.
+   */
+  std::vector<typename DoFHandler<dim>::active_cell_iterator>
+  find_cells_around_cell(
+    const typename DoFHandler<dim>::active_cell_iterator &cell);
+
+  /**
+   * @brief
+   *Clear all the line of dof even if the dof is not owned but it is ghost
+   *
+   * @param cell , The initial cell from which we access the DOF,
+   *
+   * @param dof_index , The dof index for which we want to clear the line in the matrix.
+   *
+   */
+  void
+  clear_line_in_matrix(
+    const typename DoFHandler<dim>::active_cell_iterator &cell,
+    unsigned int                                          dof_index);
+
+
+  // Modified version of assemble_matrix_and_rhs to include the presence of
+  // extra steps. For more detail see the same function in the
+  // gls_navier_stokes.h solver.
```

Comment:
Use /* */ syntax for multiline comments.

Reply:
Those comments in .cc must be from // to /* */ ?

Reply:
Yeah it is better to use /* than //